style.css

```diff
@@ -0,0 +1,119 @@
+
+body{
+    background-color:#9B90A0;
+    font-family:"Veranda", sans-serif;
```

Comment:
Note: your computer has "Veranda" installed as a default font, but it's possible that someone else might not have it in which case this will fall back to their default sans-serif font. If you want to use a non-standard font and make sure that the audience sees the same thing you see, check out Google Fonts. That's a website Google hosts that will let you pick a font you like and embed it in your website.